packages/js/product-editor/src/components/block-editor/block-editor.tsx

```diff
@@ -198,8 +198,12 @@ export function BlockEditor( {
 		product
```

Comment:
[I've fixed it for the classic editor](https://github.com/woocommerce/woocommerce/pull/47762/commits/1e4523e73330f565107bcd2738a62c6150ea122d), I think it's a harmless fix. But I'm not sure about doing this for the REST API, maybe it's enough of an edge case to skip it for now?

Reply:
> But I'm not sure about doing this for the REST API, maybe it's enough of an edge case to skip it for now?

I'm fine with handling this in a follow up issue, but without it, the `_product_template_id` isn't really reliable.

This still makes me question whether we should even persist the `_product_template_id`. This is probably something to be to brought up for discussion within the team. If we can always determine the product template via inspection of the product (in other words, a match function), then it is better to not persist the value.

For now, I would be okay with the change in this PR, since the issue of persisting the template ID is larger than the changes this PR proposes.